tests/host_tools/cargo_build.py

```diff
@@ -43,7 +45,7 @@ def cargo_test(path, extra_args=""):
     path = os.path.join(path, CARGO_UNITTEST_REL_PATH)
     cmd = (
         "CARGO_TARGET_DIR={} RUST_TEST_THREADS=1 RUST_BACKTRACE=1 "
-        'RUSTFLAGS="{}" cargo test {} --all --no-fail-fast'.format(
+        'RUSTFLAGS="{}" cargo test --locked {} --all --no-fail-fast'.format(
```

Comment:
hmm would it be possible to do it from a cargo wrapper?

```python
def cargo(cmd, env=None):
    return run_cmd("cargo --locked", cmd, env)
```

Reply:
Mh, just checked that its indeed possible to run cargo subcommands as `cargo --locked check` for instance. I think this would be beneficial because it also reduces the probability of later added tests running without the `--locked`. I'll add it when I find some time to cycle back to this, sadly some higher priority things have come up right now :(

Reply:
done, please have another look :)